tests/functional/s3cmd/tests.js

```diff
@@ -111,6 +111,27 @@ function checkRawOutput(args, lineFinder, testString, cb) {
     });
 }
 
+// Test stderr against expected output
+function checkErrOutput(args, lineFinder, testString, cb) {
```

Comment:
instead of creating a new function, perhaps add an argument to the checkRawOutput function above that tells it to check stderr instead.